en/guidance_updates.rst

```diff
@@ -0,0 +1,100 @@
+
+The `IATI Standard Publishing Guidance <https://iatistandard.org/en/guidance/standard-guidance/>`__ helps address this, by adding context and filling in some of the gaps in the Standard. These pages fit in the **“non-normative”** content category as described below.
+
+- Normative: `Normative content <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__ is the prescriptive part of a standard. It sets the rules to be followed in order to be compliant with the standard, and from which no deviation is permitted.
+
+- Non-normative: Non-normative content is the non-prescriptive, or ‘descriptive’, part of a standard. This may include explanations, illustrations, context, and examples. In the event non-normative content contradicts normative content, the normative content is to be followed.
+
+For a full list of what falls under the normative and non-normative category as part of the IATI standard, see `here <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__.
+
+Suggesting changes to the IATI Standard Publishing Guidance
+------------------------------------------------------------
+
+Normative remains the same - changes are made following a clear process and sufficient notice, to ensure relevant stakeholders are involved. The `IATI upgrade process <https://iatistandard.org/en/iati-standard/upgrades/#>`__ presently assures this.
+
+Changes to non-normative content, such as the IATI Standard Publishing Guidance, can happen without triggering an upgrade governance process, and can be updated outside of a formal upgrade process, ensuring that changes do not conflict with normative content.
+
+Changes to the `IATI Standard Publishing Guidance <https://iatistandard.org/en/guidance/standard-guidance/>`__ can either be proposed by the Secretariat (e.g. in response to specific user feedback or changes needed to deliver on the IATI Strategic Plan) and then consulted with the IATI community, or proposed by the IATI community and then reviewed and considered by the Secretariat. The process for the community to propose changes to be reviewed and considered by the Secretariat is provided below.
+
+How to suggest changes to existing guidance pages
+-------------------------------------------------
+
+All IATI Publishing Guidance pages are stored and managed in the :doc:`IATI-Guidance Github repository <IATI-Upgrades>`. If you notice an error, or would like to add a comment about a guidance page, you can raise a `GitHub issue <https://github.com/IATI/IATI-Guidance/issues/new>`__. Any issues added by community members directly to Github will be reviewed **bi-weekly** by the IATI Technical Team and labelled a *‘bug’*, *‘clarification’* or *‘discussion’*. If users are not familiar with Github and would still like to comment or raise an issue, they can contact the Technical Team via the IATI Helpdesk (support@iatistandard.org) in the first instance. The Technical Team will ensure that your issue/feedback has been addressed and shared publicly.
```

Comment:
@amy-silcock This :doc: link doesn't work:

```
/home/alex/git/IATI-Reference-Generator/IATI-Upgrades/en/guidance_updates.rst: WARNING: document isn't included in any toctree
```
Do you want it to link to https://github.com/IATI/IATI-Upgrades instead?